libinstall/CentWeb.sh

```diff
@@ -327,15 +346,19 @@ $INSTALL_DIR/cinstall $cinstall \
     $INSTALL_DIR_CENTREON/www >> "$LOG_FILE" 2>&1
 
 $INSTALL_DIR/cinstall $cinstall_opts \
-    -u "$CENTREON_USER" -g "$CENTREON_GROUP" -d 755 -m 644 \
+    -u "$CENTREON_USER" -g "$CENTREON_GROUP" -d 775 -m 775 \
```

Comment:
Wow ! Permission changes on files seem excessively permissives !

Reply:
it's needed for the debian install

Reply:
Could you please explain what is the specific issue with the existing permissions ?

Reply:
the .htaccess is modified somewhere else later... I can't find where.
But if I set it to 644, I'm stuck on the login screen and I can't access the IHM.